app/assets/javascripts/placeholder_input_size.js

```diff
@@ -0,0 +1,12 @@
+function setInputSize(id) {
+  var length = $(id).attr("placeholder").length;
+  $(id).attr("size", length);
```

Comment:
this is a little unfortunate but if you don't want to just make the input box bigger all the time, it'll do the trick...